src/module.c

```diff
@@ -1120,6 +1120,26 @@ void swapDbEventCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub,
     }
 }
 
+int persistence_in_progress = 0;
```

Comment:
@OfirMos do we use an integer because we are expecting to have more than 1 persistence flow at the same time?

Reply:
@danni-m Yes, I'm not expecting it to grow more than 1 but it's for safeties